content/data-reporting/reports-api/v3.3/index.mdx

```diff
@@ -56,16 +56,17 @@ Shows the average chatting duration of agents within a license.
 
 #### Request
 
-| Query String          | Required | Notes                                                                                                    |
-| --------------------- | -------- | -------------------------------------------------------------------------------------------------------- |
-| `to`                  | yes      | Date in the RFC3339 format                                                                               |
-| `from`                | yes      | Date in the RFC3339 format                                                                               |
-| `agents`              | no       | Agent emails separated by a comma; if not specified, returns the data for all agents within the license. |
-| `groups`              | no       | Group IDs separated by a comma                                                                           |
-| `tags`                | no       | Names of tags separated by a comma                                                                       |
-| `customer_client_ids` | no       | Client IDs separated by a comma                                                                          |
-| `distribution`        | no       | Possible values: `hour`, `day-hours`, `day`, `month`; defaults to `day`                                  |
-| `tagged`              | no       | Possible values: `true`, `1`, `false`, `0`                                                               |
+| Query String          | Required | Notes                                                                                                                                                                                                     |
+| --------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `to`                  | yes      | Date in the RFC3339 format. If `timezone` provided, the Timezone from this format will be ignored.                                                                                                        |
```

Comment:
```suggestion
| `to`                  | yes      | Date in the RFC3339 format, which also contains a timezone. This timezone will be used if no `timezone` is provided. |
```

Reply:
I rephrased the sentence a little. WDYT? It's easier to understand, IMO.

Reply:
It's nice, thanks!